power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx

```diff
@@ -0,0 +1,232 @@
+               {choices.map( x =>
+                  <label key={x} class="checkbox">
+                     <input
+                        type={type}
+                        name={inputName}
+                        value={x}
+                        required={type === "radio" ? true : false}
+                     />{" "}{x}
+                  </label> )}
+            </div>
+         </div>
+      );
+
+/**
+ * Additional info be collected after users sign up
+ */
+const AdditionalInfoForm: FunctionalComponent<{ data: PtpFormData, onSubmit: () => void }> = ( { data, onSubmit } ) => {
+
+   const submitForm = ( e: Event ) => {
+      try {
+         // gather up all the form data
+         const form = e.target as HTMLFormElement;
+         const elements = [
+            ...form.getElementsByTagName( "input" ),
+            ...form.getElementsByTagName( "select" ),
+         ];
+         const toActionKit = elements.reduce(
+            ( x, el ) => {
+               let add = false;
+
+               // make sure radio and checkboxes are checked before adding their values
+               if( el instanceof HTMLInputElement && ( el.type === "radio" || el.type === "checkbox" ) ) {
+                  if( el.checked ) {
+                     add = true;
+                  }
+               } else {
+                  add = true;
+               }
+
+               if( add ) {
+                  x[el.name] = el.name in x ? x[el.name] + ", " + el.value : el.value;
+               }
+
+               return x;
+            },
+            ( {
+               // ActionKit will clear the city, state, and zip when we update `address1`, so we have to pass them
+               // through as well. They used to be hidden inputs in the form, but this is more explicit and cleaner.
+               city: data?.city,
+               state: data?.state,
+               zip: data?.zip,
+               // this acts as the ID/lookup for the user we're modifying
+               email: data?.email,
+               // Needs to match to a Page in ActionKit or it will return 400
+               page: "mi-extra",
+            } as any ),
+         );
+
+         submitToActionKit( toActionKit )
+            .then( result => {
+               if( result !== true ) {
+                  console.error( result );
+               }
+            } );
+
+         onSubmit();
+      } catch( e ) {
+         console.error( e );
+      } finally {
+         e.preventDefault();
+         return false;
+      }
+   };
+
+   return ( <Fragment>
+
+      <p>
+         We are sharing your information with election administrators and our state partners who will follow up to help you be placed as a poll worker!&nbsp;
+        <strong>We just need a few more pieces of information from you to help with your application:</strong>
+      </p>
+
+      <hr />
+
+      <form onSubmit={submitForm} style={{ padding: "0" }}>
+
+         {data.state === "MI"
```

Comment:
could simplify to  by using logical AND `{data.state ==== "MI" && <label>`

Reply:
I wanted to leave a placeholder for other states